### PR TITLE
refactor(select): use line ripple

### LINKED ISSUE #2129: Update Select to use line ripple
Depends on #1981

### PR DESCRIPTION
**Pull request by moog16**
fixes #2129

### CODE REVIEW

File: packages/mdc-select/_mixins.scss [hunk 1]
```diff
@@ -76,18 +82,26 @@
 }
 
 @mixin mdc-select-bottom-line-color_($color) {
-  .mdc-select__bottom-line {
-    @include mdc-select-bottom-line-fill-color($color);
+  .mdc-line-ripple {
```
Comment (lynnjepsen) [inline packages/mdc-select/_mixins.scss]: I dont think you need to set the mdc-line-ripple here

File: packages/mdc-select/_mixins.scss [hunk 2]
```diff
@@ -76,18 +82,26 @@
...
+}
+
+@mixin mdc-select-focused-line-ripple-color_($color) {
+  @include mdc-select-focused-line-ripple_ {
+    @include mdc-line-ripple-color($color);
   }
 
-  .mdc-select__bottom-line::after {
-    @include mdc-select-bottom-line-fill-color($color);
+  .mdc-line-ripple::after {
```
Comment (lynnjepsen) [inline packages/mdc-select/_mixins.scss]: ::after?? whats this for?

File: packages/mdc-select/constants.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const numbers = {
 
 export const strings = {
   CHANGE_EVENT: 'MDCSelect:change',
-  BOTTOM_LINE_SELECTOR: '.mdc-select__bottom-line',
+  BOTTOM_LINE_SELECTOR: '.mdc-line-ripple',
```
Comment (lynnjepsen) [inline packages/mdc-select/constants.js]: rename this to LINE_RIPPLE_SELECTOR?

File: test/unit/mdc-floating-label/mdc-floating-label.test.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ import {assert} from 'chai';
 import {MDCFloatingLabel} from '../../../packages/mdc-floating-label/index';
 
 const getFixture = () => bel`
-  <div class="mdc-floating-label"></div>
+  <label class="mdc-floating-label"></label>
```
Comment (lynnjepsen) [inline test/unit/mdc-floating-label/mdc-floating-label.test.js]: I think this change snuck in here

File: packages/mdc-select/_mixins.scss (hunk 2 above)
Comment (moog16) [reply packages/mdc-select/_mixins.scss]: Not sure what it was even doing before. Removing the rule.

File: test/unit/mdc-floating-label/mdc-floating-label.test.js (hunk 4 above)
Comment (moog16) [reply test/unit/mdc-floating-label/mdc-floating-label.test.js]: This is done on purpose. You're right in that it is more related to the floating label PR, but it wasn't caught until doing changes with line-ripple.

**Pull request merged**